packages/dds/tree/src/test/feature-libraries/modular-schema/discrepancies.spec.ts

```diff
@@ -42,6 +42,11 @@ function validateStrictSuperset(view: TreeStoredSchema, stored: TreeStoredSchema
 	assert.equal(allowsRepoSuperset(defaultSchemaPolicy, view, stored), false);
 }
 
+// Arbitrary schema names used in tests
+const stringName = brand<TreeNodeSchemaIdentifier>("string");
```

Comment:
Since the root field used in many of these tests now need to refer to branded identifiers for all the types instead of using any, I factored them out for easy use without having to brand them everywhere.